stdlib/public/Differentiation/DifferentiationSupport.swift

```diff
@@ -212,39 +212,36 @@ public func withoutDerivative<T, R>(at x: T, in body: (T) -> R) -> R {
 /// Create a differentiable function from a vector-Jacobian products function.
 @inlinable
 public func differentiableFunction<T : Differentiable, R : Differentiable>(
-  from vjp: @escaping (T)
+  from vjp: __owned @escaping (T)
```

Comment:
I wonder if `__owned` convention is necessary to use `Builtin.differentiableFunction`?

Reply:
No, it’s not necessary. It’s just a library optimization.

Reply:
Well, I removed it because there's a bug that causes it not to parse when it's used with `@escaping`.